common/internal-types/src/tickets.rs

```diff
@@ -586,8 +589,7 @@ impl Ticket {
     /// must be equal to on-chain computation
     pub fn get_hash(&self, domain_separator: &Hash) -> Hash {
         let ticket_hash = Hash::create(&[self.encode_without_signature().as_ref()]); // cannot fail
-                                                                                     // This contains on-chain prefix: RedeemTicketCall::selector() and zeroes
-        let hash_struct = Hash::create(&[&[252u8, 183u8, 121u8, 111u8], &[0u8; 28], ticket_hash.as_ref()]);
+        let hash_struct = Hash::create(&[&redeemTicketCall::SELECTOR, &[0u8; 28], ticket_hash.as_ref()]);
```

Comment:
The `redeemTicketCall::SELECTOR` was removed on purpose, so that the `hopr-internal-types` crates does not depend on `bindings` just for this single constant.

It makes build of this crate much faster.

Reply:
Oh I see. I'll also need to document it somewhere in case we forget to update it if the interface of this particular function gets changed in SC v4.0